lib/ui/channel_buffers.dart

```diff
@@ -380,6 +380,7 @@ class ChannelBuffers {
     assert(!name.contains('\u0000'), 'Channel names must not contain U+0000 NULL characters.');
     final _Channel channel = _channels.putIfAbsent(name, () => _Channel());
     channel.setListener(callback);
+    channelListenedTo(name, true);
```

Comment:
To be consistent with the embedder API, can we rename `channelListenedTo` to `sendChannelUpdate` and `onChannelListenedTo` to `onChannelUpdate` everywhere?